webapp/src/Controller/API/GeneralInfoController.php

```diff
@@ -71,6 +73,8 @@ public function getVersionAction(): array
 
     /**
      * Get information about the API and DOMjudge
+     *
+     * @return array<string, string|array<string, string>>
```

Comment:
Array shape or DTO.

The advantage of the DTO btw is that we can also use it in the `OA\Response` I think.